BUD-2383 Updates for team budget feature

aparnasharma285 opened the pull request:
Budget is releasing new feature called team budgets in August end. These changes are for the same.

aparnasharma285 committed:
BUD-2383 Updates for team budget feature

jonikamelcher commented:
HI @aparnasharma285. Have these changes been through the CPA review? In order to release this we would need approval from them and a release note announcing the changes.

aparnasharma285 commented:
Hi @jonikamelcher no it hasn't been through CPA review. I am not sure about the process. Could you point me the right team to reach out to.

jonikamelcher commented:
@aparnasharma285 Alexei Potiagalov should be able to help you with the review, and may connect you with Alyson Nawossa for the launch process.

aparnasharma285 committed:
Update text as per review comments

aparnasharma285 committed:
Add team budget example for get budget call

Adding an example to show the response of getting a team budget

alexeipo commented on src/api-reference/budget/v4.budget-header.md:
```diff
@@ -817,6 +1255,111 @@ Authorization: Bearer {token}
+          "email": "meenapriya.meenakshisundaram@sap.com",
+          "externalUserCUUID": "8000011190000003453"
+      },
+      "budgetManagers": [],
```
If no values should be passed for these fields, then I recommend to change the example to remove the fields from the payload assuming the API works that way.

aparnasharma285 replied:
Changes done

alexeipo reviewed (commented):
The change adds a new attribute. Therefore, a review is optional.

aparnasharma285 committed:
Remove approvers & managers from example

Since approvers and managers are not required for team budgets, that can be moved from the example.

aparnasharma285 commented:
@jonikamelcher  Could you merge this PR please.

jonikamelcher commented:
Hi @aparnasharma285. Before this can be merged or released, we will need to publish a release note for this detailing the new functionality. Please submit a release note to  DL Concur_DevCenter <DL_5BA96883ECB2111D8C00004C@global.corp.sap>. Thank you.

aparnasharma285 committed:
Update field name

anuvenkataraman committed:
Update v4.budget-adjustments.md

Updating information for date range budgets

aparnasharma285 committed:
Merge pull request #1 from anuvenkataraman/patch-1

Update v4.budget-adjustments.md

jonikamelcher commented:
:shipit:
